controller/workitem.go

```diff
@@ -524,6 +526,25 @@ func ConvertJSONAPIToWorkItem(ctx context.Context, method string, appl applicati
 	return nil
 }
 
+// replaceFieldNames modifies the source to rename all system.* fields to
+// system_* fields. It uses the OldToNewFieldNameMap to find the correct new
+// name for the old field
+// TODO(ibrahim) - Remove these after fields are migrated
```

Comment:
Please use your GH handle in TODOs: `TODO(jarifibrahim): ...`.

Reply:
I like `ibrahim` more than `jarifibrahim` :wink: 
Is there a reason to use GH handle instead of name?
I guess it's okay to have name/GH handle.

Reply:
The reason is that it identifies you.

Reply:
Yup. `ibrahim` and `jarifibrahim` is the same person :)